Packages/vcs/Lib/VCS_validation_functions.py

```diff
@@ -877,6 +883,10 @@ def checkLineType(self, name, value):
         hvalue = 'long-dash'
     elif (queries.isline(value) == 1):
         hvalue = value.name
+    elif value in vcs.elements["line"]:
+        self.linecolor = vcs.elements["line"][value].color[0]
```

Comment:
yes it should never happen vcs.elements are auto-populated when creating vcs objects